data_prep/introspector.py

```diff
@@ -747,7 +755,8 @@ def _parse_arguments() -> argparse.Namespace:
   parser.add_argument('-t',
                       '--target-oracle',
                       type=str,
-                      default='far-reach-low-coverage',
+                      nargs='+',
+                      default=['optimal-targets', 'all-public-candidates'],
```

Comment:
I'd keep `far-reach-low-coverage` as the primary benchmark -- there will still be improvements with respect to build rate from the use of `exclude-static-functions=True&only-referenced-functions=False&only-with-header-file-declaration=True`

`optimal-targets` is a bit more intricate and has far fewer functions per project. For some there may be no due to additional filtering from `exclude-static-functions=True&only-referenced-functions=False&only-with-header-file-declaration=True`

Reply:
I see, I will use `['optimal-targets', 'far-reach-low-coverage']` then.

> all-public-candidates is primarily a JVM oracle. 

Could you please reflect that in the name of `INTROSPECTOR_ORACLE_ALL_CANDIDATES`, like `INTROSPECTOR_ALL_JVM_SOURCE_PATH`?